### PR TITLE
[DRGN-428] refactor: update schema and remove dead code

### PR DESCRIPTION
**Pull request by rdaquilante**
DRGN-428

### CODE REVIEW

File: src/interfaces/DragonchainClientInterfaces.ts [hunk 1]
```diff
@@ -147,18 +145,24 @@ export interface DragonchainBulkTransactionCreateResponse {
...
   'dcrn': 'SmartContract::L1::AtRest',
-  'version': '1' | '2',
+  'version': '3',
+  'txn_type': string,
   'id': string,
-  'name': string,
-  'status': 'approved' | 'rejected' | 'pending',
-  'custom_environment_variables': object,
-  'origin': 'library' | 'custom',
-  'runtime': ContractRuntime,
```
Comment (DRigby26) [inline src/interfaces/DragonchainClientInterfaces.ts]: Do we no longer need to specify runtime?

File: src/interfaces/DragonchainClientInterfaces.ts [hunk 2]
```diff
@@ -14,8 +14,6 @@
  * limitations under the License.
  */
 
-export type ContractRuntime = 'nodejs6.10' | 'nodejs8.10' | 'java8' | 'python2.7' | 'python3.6' | 'dotnetcore1.0' | 'dotnetcore2.0' | 'dotnetcore2.1' | 'go1.x'
```
Comment (DRigby26) [inline src/interfaces/DragonchainClientInterfaces.ts]: Do we no longer need to specify runtime?

Comment (rdaquilante) [reply src/interfaces/DragonchainClientInterfaces.ts]: Runtime doesn't exist for docker images. The docker image itself downloads and runs whatever code is necessary. This was a remnant left from the days of lambda.

Comment (DRigby26) [reply src/interfaces/DragonchainClientInterfaces.ts]: Ah gotchya. Cool!

**Pull request merged**